config/prow/autobump-config/prow-component-autobump-config.yaml

```
@@ -2,7 +2,7 @@
 gitHubLogin: "k8s-ci-robot"
 gitHubToken: "/etc/github-token/oauth"
 gitName: "Kubernetes Prow Robot"
-gitEmail: "k8s.ci.robot@gmail.com"
+gitEmail: "github+k8s-infra-ci-robot@kubernetes.io"
```

```suggestion
gitEmail: "github+k8s-ci-robot@kubernetes.io"
```

We have multiple bots and each one use now a alias for `github@kubernetes.io`. 🤷🏾